presto/auth.py

```diff
@@ -85,7 +85,7 @@ def set_http_session(self, http_session):
             delegate=self._delegate,
             service=self._service_name,
         )
-        if self._ca_bundle:
+        if self._ca_bundle is not None:
```

Comment:
Is your intention to have _ca_bundle=False here so that `http_session.verify` will be set to `False`?

Reply:
Yes. Exactly.

Reply:
Does something like how PostgreSQL (and others) construct their URLs make sense? i.e. `?ssl=true&sslmode=verify&sslcert=cert.crt`

`ssl` (http/https) and `sslcert` (ca bundle) are already there.
`sslmode` can be introduced as a new parameter.